docs/api-reference/instances/index.md

```diff
@@ -37,6 +37,23 @@ Utilities for connecting state objects to instances via code.
 
 <div class="fusiondoc-index-multicol-section" markdown>
 ### Functions
+<a class="fusiondoc-api-index-link" href="attribute" markdown>
+	<span class="fusiondoc-api-icon" markdown>:octicons-code-24:</span>
+	<span class="fusiondoc-api-name">Attribute</span>
+    <span class="fusiondoc-api-index-arrow" markdown>:octicons-chevron-right-24:</span>
+</a>
+
+<a class="fusiondoc-api-index-link" href="attributeChange" markdown>
```

Comment:
You'll need to update these links if you rename the files, so I'll comment here as a reminder :)

Reply:
Thank you!